dmipy/core/modeling_framework.py

```diff
@@ -1212,8 +1216,24 @@ def fit(self, acquisition_scheme, data,
             start = time()
             mt_fractions = np.empty(
                 np.r_[N_voxels, self.N_models], dtype=float)
+
+            S0_responses = []
+            for i, (m, s) in enumerate(zip(self.models,
```

Comment:
Am i wrong or is `i` not being used at all?

Reply:
You're right. It will be used once `S0_responses` will be defined as a numpy array